cli/ssh.go

```diff
@@ -42,13 +42,15 @@ var (
 	autostopNotifyCountdown = []time.Duration{30 * time.Minute}
 )
 
+//nolint:gocyclo
 func (r *RootCmd) ssh() *clibase.Cmd {
 	var (
 		stdio          bool
 		forwardAgent   bool
 		forwardGPG     bool
 		identityAgent  string
 		wsPollInterval time.Duration
+		wait           bool
 		noWait         bool
```

Comment:
Same as with https://github.com/coder/coder/pull/7893#discussion_r1221737760 - we could potentially (breaking-ly) refactor this to `--wait=(always|never|auto)`